### PR TITLE
imported TRAI counter comment PDF and updated pubs page per #253

### PR DESCRIPTION
**Pull request by critzo**
This PR imports our counter comment to TRAI and adds it to the pubs page.
Previewable here: https://critzo.github.io/m-lab.github.io/publications/#government--regulatory-filings

---
This change is [Reviewable](https://reviewable.io/reviews/m-lab/m-lab.github.io/254)

### CODE REVIEW

File: _pages/07-publications.md [hunk 1]
```diff
@@ -385,7 +385,12 @@ M. Dischinger, A. Mislove, A. Haeberlen, K. P. Gummadi
 ## 2017
 {:.no_toc}
 
-### [4/12/2017 - TRAI (India)]({{ site.baseurl }}/publications/regulator-filings/ntia/m-lab_ntia_10-11-2016.pdf){:target="_blank"}: TRAI Consultation Paper on Net Neutrality
+### [4/25/2017 - TRAI (India)](http://trai.gov.in/sites/default/files/Measurement_Lab_27_04_2017.pdf){:target="_blank"}: Counter Comments on TRAI Consultation Paper on Net Neutrality
```
Comment (collina) [inline _pages/07-publications.md]: There's no need to link out to TRAI. We might as well have all our links be M-Lab.

Comment (critzo) [inline _pages/07-publications.md]: Linking to the regulator is what we've done for the FCC links, so I'm inclined to do it here as well. @georgiamoon what do you think?

Comment (georgiamoon) [inline _pages/07-publications.md]: @collina is your concern that that link might change? To this point, we could have the links be to the source documents, and the "Download" links be local to m-lab hosting.

**Comment by georgiamoon** [comment]
LGTM.

**Pull request merged**